src/cleaner_utils.cpp

```diff
@@ -25,6 +28,8 @@ FunctorSequenceCleaner::FunctorSequenceCleaner(GlobalSequencesViewer& globalSequ
 
 void FunctorSequenceCleaner::cleanup() {
     globalSequencesViewer.removeIdleSequences();
+    SPDLOG_TRACE("malloc_trim(0)");
```

Comment:
Maybe something more meaningfull ? Calling malloc_trim(0) in sequence cleaner.

Reply:
I would leave it as is, it is trace anyway so we will most likely grep it